src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__cell cell cell--1 cell--tue"></div>
```

Comment:
```suggestion
      <div class="calendar__cell cell"></div>
```
it should be modificators of the calendar:
![image](https://user-images.githubusercontent.com/86923089/218547425-2288fceb-ce83-475c-a8d3-47908ccc022a.png)